modules/release-notes/pages/index.adoc

```diff
@@ -3,52 +3,77 @@
 
 === Security
 * Enhanced the xref:user-access:access-control-model.adoc#_role_based_access_control[role-based access control] system to allow data CRUD privileges to be granted at the type and attribute level (Preview).
+** Local roles are deprecated as of the 3.7 release and will be dropped in a future version.
 * Added the ability to enable a xref:security:password-policy.adoc[password policy] to mandate the minimal complexity of passwords.
 * TigerGraph Server download packages are now xref:security:index.adoc#_pgp_key[signed with our PGP key].
 
 === Ecosystem integration
-* Added the ability to xref:data-loading:data-streaming-connector/azure-blob.adoc[stream data from Azure Blob Storage] to xref:data-loading:data-streaming-connector/index.adoc[Data Streaming Connector].
-* Added the ability to xref:data-loading:data-streaming-connector/aws-s3.adoc[stream data from ]for all supported sources with xref:data-loading:data-streaming-connector/index.adoc[Data Streaming Connector].
+* Added the ability to xref:data-loading:data-streaming-connector/azure-blob.adoc[stream data from Azure Blob Storage] to a xref:data-loading:data-streaming-connector/index.adoc[Data Streaming Connector].
+* Added the ability to xref:data-loading:data-streaming-connector/aws-s3.adoc[stream data from AWS S3] to xref:data-loading:data-streaming-connector/index.adoc[Data Streaming Connector].
 * Deleting a Data Streaming Connector now deletes the data in the topics created by the connector.
 
-=== Performance
-* Introduced the cost-based query optimizer (Preview), which has the potential to significantly reduce query execution time.
-
 === Observability
-
-
-=== Data API service
+* Improved logging for privilege checks.
+Every time a user performs an action that requires any privilege, a detailed log of the action and the user is generated.
+** You must xref:user-access:enabling-user-authentication.adoc[enable authentication] for privilege checking to happen.
+* GSQL command to display loading job status (`SHOW LOADING STATUS`) now supports all loading jobs, including loading jobs created with xref:data-loading:data-streaming-connector/index.adoc[] and xref:data-loading:kafka-loader/index.adoc[Kafka Loader].
 
 
 === Query Language Enhancement
-* Token functions for `WHERE` clause can now also be used for attribute expressions.
+* xref:gsql-ref:ddl-and-loading:functions/token/index.adoc[Token functions for attribute expressions] now also be used in expressions in the `WHERE` clause.
 
 
 
 [#_fixed_issues]
 == Fixed issues
+* Fixed a number of security vulnerabilities from package dependencies.
+* Improved error messaging when a query uses `TYPEDEF TUPLE` with an unsupported data type.
+* Fixed an issue where unauthorized users could access certain local files on the server through a REST endpoint.
+* Fixed an issue where running multiple resource-intensive queries simultaneously causes would cause GPE dysfunction due to critical memory level handling.
```

Comment:
```suggestion
* Fixed an issue where running multiple resource-intensive queries simultaneously would cause GPE dysfunction due to critical memory level handling.
```